examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml

```diff
@@ -0,0 +1,259 @@
+# It contains the default values for training a Conformer-Transducer ASR model with stateless decoder, large size (~120M) with Transducer loss and sub-word encoding.
```

Comment:
Could you add this to NeMo doc?  such as [here]( https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-transducer )